integration-tests/optaplanner/pom.xml

```diff
@@ -95,6 +132,42 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
I created this JIRA : https://issues.redhat.com/browse/PLANNER-2149

Reply:
it would be fixed at some point, meanwhile I can't run it with enforcer

Reply:
I can deal with it by exclusion and setting the right mvel2 version, but we have to be careful for future versions of optaplanner. Or wait next optaplanner release to add this extension. I don't know what's the best solution?

Reply:
> I can deal with it by exclusion and setting the right mvel2 version

That's probably better. If we skip the enforcer plugin, then it potentially hides other issues that might appear that are unrelated to mvel.

We can create an issue to track removal of the exclusion etc when a fixed optaplanner is available. [PLANNER-2173](https://issues.redhat.com/browse/PLANNER-2173) seems to suggest the fix will be in 7.45.

There is no rush to merge this since Camel 3.6.0 is not GA. Maybe by the time that it is GA, 7.45 will be ready.